subsys/esb/esb_ppi_api.h

```diff
@@ -90,6 +90,14 @@ void esb_ppi_for_wait_for_ack_set(void);
  */
 void esb_ppi_for_wait_for_ack_clear(void);
 
+/** @brief Configure PPIs/DPPIs for an a packet without ACK and with never_disable_tx ON.
+ */
+void esb_ppi_for_wait_for_rx_set(void);
+
+/** @brief Clear PPIs/DPPIs configuration for a packet without ACK and with never_disable_tx ON.
+ */
+void esb_ppi_for_wait_for_rx_clear(void);
```

Comment:
Where is this function used? It should be used somewhere for clearing PPI connection for this mode.

Reply:
Forgot a call...
Now called by on_radio_end_tx_noack()